backend_new/src/dtos/application-flagged-sets/afs-query-params.dto.ts

```diff
@@ -0,0 +1,22 @@
+import { ApiProperty, ApiPropertyOptional } from '@nestjs/swagger';
+import { Expose } from 'class-transformer';
+import { IsEnum, IsUUID } from 'class-validator';
+import { View } from '../../enums/application-flagged-sets/view';
+import { ValidationsGroupsEnum } from '../../enums/shared/validation-groups-enum';
+import { PaginationAllowsAllQueryParams } from '../shared/pagination.dto';
+
+export class AfsQueryParams extends PaginationAllowsAllQueryParams {
+  @Expose()
+  @ApiProperty()
+  @IsUUID(4, { groups: [ValidationsGroupsEnum.default] })
+  listingId: string;
+
+  @Expose()
+  @ApiPropertyOptional({
+    enum: View,
+    enumName: 'AfsView',
+    example: 'active',
+  })
+  @IsEnum(View, { groups: [ValidationsGroupsEnum.default] })
+  view?: View;
```

Comment:
Surprised to see the afs get endpoint fail (Bad Request) without a view selected even though its marked as optional here
<img width="1440" alt="Screenshot 2023-10-13 at 11 46 35 AM" src="https://github.com/bloom-housing/bloom/assets/53269332/cd1f71fa-8d31-4f81-a3ff-1a9177b9346b">

Reply:
So view in this case isn't always passed from the front end, so I think making it required would be kinda bad 

I'm not sure what change I made that seems to have fixed this, but it seems like its fixed from what I can tell